__tests__/server/ssrServer.spec.js

```diff
@@ -393,80 +437,58 @@ describe('ssrServer', () => {
       });
 
       it('should parse forwarded header for render post calls', async () => {
-        await request(loadServer())
+        await request(loadExpress())
           .post('/route');
         expect(forwardedHeaderParser).toBeCalled();
       });
     });
 
     it('should log the csp violation request body', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .post('/_/report/security/csp-violation')
         .send({ body: 'foo', other: 'boo' });
       expect(cspViolation).toHaveBeenCalled();
     });
 
     it('should create send html for all get calls', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(sendHtml).toBeCalled();
     });
 
     it('should not call route specific middleware to be called on non-matching routes', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(cspViolation).not.toHaveBeenCalled();
       expect(clientErrorLogger).not.toHaveBeenCalled();
     });
 
     it('should create request html fragment for all get calls', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(createRequestHtmlFragment).toBeCalled();
     });
 
     it('should check the state for a redirect for all get calls', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(checkStateForRedirect).toBeCalled();
     });
 
     it('should check the state for a status code for all get calls', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(checkStateForStatusCode).toBeCalled();
     });
 
     it('should add frame options header for all get calls', async () => {
-      await request(loadServer())
+      await request(loadExpress())
         .get('/route');
       expect(addFrameOptionsHeader).toBeCalled();
     });
 
-    it('should return the static error page when an error was encountered', async () => {
```

Comment:
is this still being tested elsewhere ?

Reply:
it's covered by fastify but I added it back